frontend/viewer/src/project/ProjectSidebar.svelte

```diff
@@ -33,44 +33,38 @@
 </script>
 
 {#snippet ViewButton(view: View, icon: IconClass, label: string)}
-  <Sidebar.MenuSubItem>
-    <Sidebar.MenuSubButton onclick={() => (currentView = view)} isActive={currentView === view}>
+  <Sidebar.MenuItem>
+    <Sidebar.MenuButton onclick={() => (currentView = view)} isActive={currentView === view}>
```

Comment:
Or was there a reason you used SubItems?
Unless we're planning on making it collapsible, this seems fine.
But, we can revisit it any time.

Reply:
Yeah I wanted it to be nested under the dictionary section, I think having it be a sub item made it indented

Reply:
though actually we might want a sub item under tasks... so maybe this is fine?

Reply:
OK. I'm gonna merge this. We can keep talking about it.